w08h02/test/pgdp/pools/TaskPoolBehaviorTest.java

```diff
@@ -51,6 +51,21 @@ void artemisExample4() {
         assertEquals(task, pool.getByValue(1, func), "Querying a Task with getByValue which was added before should yield that task.");
     }
 
+    @Test
+    @DisplayName("Check if Compare works right")
+    void compareCheck(){
```

Comment:
```suggestion
    void compareCheck() {
```

Nit: Just code styling